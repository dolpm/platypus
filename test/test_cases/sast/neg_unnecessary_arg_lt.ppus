pipe test |> [] |> [x: &'a int] |> unit {
  |> ();
}

pipe main |> [] |> [] |> unit {
  |> ();
}
