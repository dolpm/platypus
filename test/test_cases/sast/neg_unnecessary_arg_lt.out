Fatal error: exception Failure("variable x will not be returned from test and thus its lifetime isn't required")
